### PR TITLE
[improvement](jdbc catalog) support jdbc external catalog insert stmt in nereids

### PR DESCRIPTION
**Pull request by zy-kkk**

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/properties/RequestPropertyDeriver.java [hunk 1]
```diff
@@ -152,6 +153,17 @@ public Void visitPhysicalIcebergTableSink(
         return null;
     }
 
+    @Override
+    public Void visitPhysicalJdbcTableSink(
+            PhysicalJdbcTableSink<? extends Plan> jdbcTableSink, PlanContext context) {
+        if (connectContext != null && !connectContext.getSessionVariable().enableStrictConsistencyDml) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/properties/RequestPropertyDeriver.java:159]: i think jdbc sink should always use gather properties

File: fe/fe-core/src/main/java/org/apache/doris/nereids/analyzer/UnboundTableSinkCreator.java [hunk 2]
```diff
@@ -110,6 +114,9 @@ public static LogicalSink<? extends Plan> createUnboundTableSinkMaybeOverwrite(L
         } else if (curCatalog instanceof IcebergExternalCatalog && !isAutoDetectPartition) {
             return new UnboundIcebergTableSink<>(nameParts, colNames, hints, partitions,
                 dmlCommandType, Optional.empty(), Optional.empty(), plan);
+        } else if (curCatalog instanceof JdbcExternalCatalog) {
+            return new UnboundJdbcTableSink<>(nameParts, colNames, hints, partitions,
+                    dmlCommandType, Optional.empty(), Optional.empty(), plan);
         }
         // TODO: we need to support insert into other catalog
         try {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/analyzer/UnboundTableSinkCreator.java:122]: remove these force fallback code

**Comment by zy-kkk** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Comment by zy-kkk** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**